src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -3344,6 +3410,11 @@ public int EndReceiveFrom(IAsyncResult asyncResult, ref EndPoint endPoint)
 
             int bytesTransferred = castedAsyncResult.InternalWaitForCompletionInt32Result();
             castedAsyncResult.EndCalled = true;
+            if (SocketsTelemetry.Log.IsEnabled())
+            {
+                SocketsTelemetry.Log.BytesReceived(bytesTransferred);
+                if (SocketType == SocketType.Dgram) SocketsTelemetry.Log.DatagramReceived();
+            }
```

Comment:
What about methods like Socket.SendFile?

Reply:
It doesn't report the number of transferred bytes and the existing NetEventSource logs only source / destination addresses and file name, but not the size. Do you know how I can retrieve the transferred byte count here?